app/src/androidTest/java/ch/epfl/sweng/eventmanager/test/repository/MockEventsRepository.java

```diff
@@ -104,15 +104,15 @@
         Uri fakeImgUri = Uri.parse("android.resource://ch.epfl.sweng.eventmanager/drawable/event_default_cover");
 
         addEvent(new Event(1, "Event with scheduled items", "Description", new Date(1550307600L), new Date(1550422800L),
-                orgaEmail, fakeImgUri, new EventLocation("EPFL", Position.EPFL), usersMap, "JapanImpact",
+                orgaEmail, fakeImgUri, new EventLocation("EPFL", Position.EPFL), usersMap, "JapanImpact", "793504527660961",
```

Comment:
Is this the Facebook token ? If yes then you should have declared it in a constant

Reply:
yes it is the facebook token

Reply:
I advise you to do so in case the token changes, it is easier not to forget having written it somewhere !

Reply:
So it is the same for "JapanImpact" !